src/backend/main.py

```diff
@@ -23,7 +23,7 @@ def main(args: Any):
 
     # check if we use cloud data or local data
     if args.path is None or args.mode is None:
-        imgs = [Mat.read(file) for file in sorted(glob.glob("pipeline/test/data/mosaicing/farm/D*.JPG"))]
+        imgs = [Mat.read(file) for file in sorted(glob.glob("pipeline/test/data/disease/*.JPG"))]
         path = "pipeline/test/data/mosaicing/farm/D*.JPG"
         cloud_config = CloudConfig(bucket_name=output_bucket)
     else:
```

Comment:
Could you move this into a test? No need for extensive testing, but we're gonna overwrite `main.py` at some point and then it will be difficult to work with disease insights again.